packages/core/src/queries/user.test.ts

```diff
@@ -236,165 +234,6 @@ describe('user query', () => {
     await expect(hasUserWithIdentity(target, mockUser.id)).resolves.toEqual(true);
   });
 
-  it('countUsers', async () => {
-    const search = 'foo';
-    const expectSql = sql`
-      select count(*)
-      from ${table}
-      where ${fields.primaryEmail} ilike $1 or ${fields.primaryPhone} ilike $2 or ${fields.username} ilike $3 or ${fields.name} ilike $4
-    `;
-
-    mockQuery.mockImplementationOnce(async (sql, values) => {
-      expectSqlAssert(sql, expectSql.sql);
-      expect(values).toEqual([`%${search}%`, `%${search}%`, `%${search}%`, `%${search}%`]);
-
-      return createMockQueryResult([dbvalue]);
-    });
-
-    await expect(countUsers(search)).resolves.toEqual(dbvalue);
-  });
```

Comment:
original query tests are removed since they are not that useful per our discussion.
added comprehensive unit tests for search utils.